keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -77,9 +79,10 @@ File and block are first class citizens within the Kubernetes ecosystem.  Object
 +  _BucketClass_ - A cluster-scoped custom resource containing fields defining the provisioner and an immutable parameter set.
    + _In Greenfield_: an abstraction of new bucket provisioning.
    + _In Brownfield_: an abstration of an existing objet store bucket.
-+ _BucketContent_ - A cluster-scoped custom resource, bound to a Bucket and containing relevant metadata. 
++ _BucketContent_ - A cluster-scoped custom resource bound to a Bucket and containing relevant metadata.
 + _Container Object Storage Interface (COSI)_ -  A specification of gRPC data and methods making up the communication protocol between the driver and the sidecar.
 + _COSI Controller_ - A central controller responsible for mananing Buckets, BucketContents, and Secrets cluster-wide.
++ _COSIDriver_ - A cluster-scoped custom resource which serves the purpose of registering a driver. It is created by the sidecar controller and guarantees unique driver names. Unlike the _CSIDriver_, it is not optional.
```

Comment:
per side-bar convo - let's rename this to COSIRegistration.  Reason:  there is no overlap in purpose between this object and a CSIDriver.